a11y-dialog.js

```diff
@@ -26,17 +29,24 @@
    * @param {Element} node
    * @param {(NodeList | Element | string)} targets
    */
-  function A11yDialog(node, targets) {
+  function A11yDialog(node, targets, params) {
     // Prebind the functions that will be bound in addEventListener and
     // removeEventListener to avoid losing references
     this._show = this.show.bind(this);
     this._hide = this.hide.bind(this);
     this._maintainFocus = this._maintainFocus.bind(this);
     this._bindKeypress = this._bindKeypress.bind(this);
 
+    var options = mergeDefaultOptions(params);
+
     // Keep a reference of the node and the actual dialog on the instance
     this.container = node;
-    this.dialog = node.querySelector('dialog');
+    this.dialog =
+      node.querySelector('dialog') ||
```

Comment:
We could do `node.querySelector('dialog, [data-a11y-dialog-element]')` here to save some code.